test/src/config.ts

```diff
@@ -1,7 +1,6 @@
 import { SDKInitConfig } from "../../src/sdkRuntimeModels";
 
 export const urls = {
-    eventsV2: 'https://jssdks.mparticle.com/v2/JS/test_key/Events',
```

Comment:
Nitpick: Since we're getting rid of v2, should we rename `eventsV2` to be `events`?